packages/twenty-front/src/modules/activities/emails/right-drawer/components/RightDrawerEmailThread.tsx

```diff
@@ -4,11 +4,11 @@ import { useRecoilValue } from 'recoil';
 
 import { EmailThreadHeader } from '@/activities/emails/components/EmailThreadHeader';
 import { EmailThreadMessage } from '@/activities/emails/components/EmailThreadMessage';
+import { FetchMoreLoader } from '@/activities/emails/components/FetchMoreLoader';
```

Comment:
name it EmailThreadFetchMoreLoader to make 100% clear it's not generic (also it will avoid naming conflict later, we avoid having two components with the same name for dev xp - auto import, file search in IDE especially)